engine/components/IgeTiledComponent.ts

```diff
@@ -94,9 +99,12 @@ export class IgeTiledComponent extends IgeComponent {
 					layersById[layer.name] = maps[i];
 					tileSetCount = tileSetArray.length;
 
-					if (this._ige.isClient) {
+					if (isClient) {
+						const m = maps[i] as IgeTextureMap
```

Comment:
Rather than single `m`, more descriptive variable name like `map` might be better